styles.css

```diff
@@ -8,111 +8,72 @@
     display: flex;
     flex-direction: row;
     align-items: center;
-    padding: 80px 165px;
     gap: 30px;
     background: #FFFFFF;
-    width: 1440px;
-    height: 332px;
+    height: 335px;
+    justify-content: center;
+    width: 100%;
 }
+
 .stats-section-firstcolumn {
     display: flex;
     flex-direction: column;
-    align-items: flex-start;
-    padding: 0px;
     gap: 8px;
-    width: 540px;
-    height: 168px;
-
-
 }
 .stats-firstcolumn-heading {
     display: flex;
     flex-direction: column;
-    align-items: flex-start;
-    padding: 0px;
     gap: 8px;
     color: #18191F;
     width: 540px;
-    height: 128px;
+    height: 130px;
     font-weight: 800;
     font-size: 48px;
  }
  .stats-firstcolumn-text {
-    width: 540px;
-    height: 32px;
     font-weight: 400;
     font-size: 18px;
     color: #18191F;
  }
 .stats-section-secondcolumn {
     display: flex;
     flex-direction: column;
-    align-items: center;
-    padding: 0px;
     gap: 40px;
-    width: 540px;
-    height: 172px;
-
-
-}
-.stats-secondcolumn-firstrow {
-    display: flex;
-    flex-direction: row;
-    align-items: center;
-    padding: 0px;
-    gap: 30px;
-    width: 540px;
-    height: 66px;
 }
-.stats-secondcolumn-secondrow {
+.stats-secondcolumn-rows {
     display: flex;
-    flex-direction: row;
-    align-items: center;
-    padding: 0px;
     gap: 30px;
     width: 540px;
```

Comment:
I want you to check through the inspect mode by removing/unchecking the width from there(inspect mode) and make a change here. The reason is that the content is overflowing out of the container. And if you can check this by "pesticide for chrome" extension if you have added it to your chrome extensions pack.